script.js

```diff
@@ -13,6 +14,212 @@ function closeNav() {
   div.classList.remove('blur');
 }
 
-document.querySelector(' #hamburger-menu').addEventListener('click', openNav);
+try {
+  document.querySelector('#openMenuBtn').addEventListener('click', openNav);
+} catch { console.log('error handled'); }
+document.querySelector('#closebtn').addEventListener('click', closeNav);
+
+const arrMobile = [
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    image: 'images/Snapshoot Portfolio (1).png',
+    technology: ['HTML', 'CSS', 'Javascript'],
+    'link-live-version': '#',
+    'link-to-source': '#',
+
+  },
+  {
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads no accounts or sign-ups required',
+    image: 'images/Snapshoot Portfolio2.png',
+    technology: ['HTML', 'CSS', 'Javascript'],
+    'link-live-version': '#',
+    'link-to-source': '#',
+
+  },
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    image: 'images/Snapshoot Portfolio3.png',
+    technology: ['HTML', 'CSS', 'Javascript'],
+    'link-live-version': '#',
+    'link-to-source': '#',
+
+  },
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    image: 'images/Snapshoot Portfolio4.png',
+    technology: ['HTML', 'CSS', 'Javascript'],
+    'link-live-version': '#',
+    'link-to-source': '#',
+  }];
+
+const backdrop = document.getElementById('backdrop');
+backdrop.style.display = 'none';
+
+function closePopup() {
+  document.getElementById('project-popup').style.display = 'none';
+  const popupContainer = document.getElementById('project-popup');
+  if (popupContainer) {
+    popupContainer.innerHTML = ''; // Clear the popup content
+  }
+  const backdrop = document.getElementById('backdrop');
+  backdrop.style.display = 'none';
+}
+
+function Popup(name, descr, image, technology, l1, l2) {
+  const backdrop = document.getElementById('backdrop');
+  backdrop.style.display = 'block';
+
+  document.getElementById('project-popup').style.display = 'block';
+
+  const parentElement = document.getElementById('project-popup');
+  const parent = document.createElement('section');
```

Comment:
- [ ] One of the core requirements of this milestone is that we follow the given design when implementing ours. Kindly do so

### Desktop Version
Your design
![image](https://github.com/umarahgul/Portfolio/assets/81932478/e12d93b6-7f83-45e1-9bb2-ec59da56d68e)




Given design
![image](https://github.com/umarahgul/Portfolio/assets/81932478/220c5145-cf52-48e7-960a-37732479a4ee)

 - your design has missing elements that are supposed to be below the card heading, kindly add them
 - The size of the card should be increased to match the given design
 - the languages should be aligned at the top in the same line with the paragraph of text to the left

Yours
![image](https://github.com/umarahgul/Portfolio/assets/81932478/faa03cac-5b18-4a1d-8798-a87747f3f3b0)

Given
![image](https://github.com/umarahgul/Portfolio/assets/81932478/9a26c803-cd5d-490c-8912-7131c4edecb8)

- The color for the heading should be `#091E42` and not the current `#172b4d;`, also use an h2 tag instead of an h1 for the heading for proper page structure


### Mobile Version 

Your Design
![image](https://github.com/umarahgul/Portfolio/assets/81932478/5a87be16-8323-49ed-8ed6-45f10a28bb74)

Given design
![image](https://github.com/umarahgul/Portfolio/assets/81932478/e906bfa3-64e7-4f69-940c-e258aa90795a)

- Kindly ensure your mobile version is fully visible in mobile view. This is because the modal has an overflow on its x-axis. The following view is how it looks if I try to bring it into view
![image](https://github.com/umarahgul/Portfolio/assets/81932478/c8c9ff27-4c0f-4314-acaa-9797cf4dabfa)
- Also for the mobile version, add the missing elements below the modal heading
- Once you get rid of the overflow kindly ensure the buttons at the bottom are aligned like in the given design